src/Identifier/OrmIdentifier.php

```diff
@@ -96,17 +98,27 @@ protected function _findUser($username, $password = null)
     /**
      * Get query object for fetching user from database.
      *
-     * @param string $username The username/identifier.
+     * @param string $identifier The username/identifier.
      * @return \Cake\ORM\Query
      */
-    protected function _query($username)
+    protected function _query($identifier)
     {
         $config = $this->_config;
         $table = TableRegistry::get($config['userModel']);
 
-        $options = [
-            'conditions' => [$table->aliasField($config['fields']['username']) => $username]
-        ];
+        if (is_array($config['fields']['username'])) {
+            $conditions = [];
+            foreach ($config['fields']['username'] as $field) {
+                $conditions[$table->aliasField($field)] = $identifier;
+            }
+            $conditions = [
+                'OR' => $conditions
+            ];
+        } else {
+            $conditions = [$table->aliasField($config['fields']['username']) => $identifier];
+        }
+
+        $options = compact('conditions');
```

Comment:
I would prefer if we keep them, the prefixing is pretty helpful IMO.

So do we want to remove them all or not? If we want to remove them all this should be done probably in another PR.